### PR TITLE
Update insert hole parameters, fix backplate attribute bug

### PR DESCRIPTION
**Pull request by rymilton**
### Briefly, what does this PR introduce?
Updated the forward insert's hole initial/final radius and x-position after the z positions of the forward endcap were changed to managerie values (329.6 cm for ECal, 359.6 for HCal).
Fixed bug where the attribute backplate_thickness wasn't being read in.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #243 )
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [x] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
None
### Does this PR change default behavior?
Adjusts default shape of the forward inserts

### CODE REVIEW

File: src/InsertCalorimeter_geo.cpp [hunk 1]
```diff
@@ -49,7 +49,7 @@ static Ref_t createDetector(Detector& desc, xml_h handle, SensitiveDetector sens
   const std::pair<double, double> hole_y_parameters(hole_y_initial, hole_y_final);
 
   // Getting thickness of backplate
-  auto backplate_thickness = dd4hep::getAttrOrDefault<double>(detElem, _Unicode(backplate_thickness), 0. * cm);
+  auto backplate_thickness = detElem.hasChild(_Unicode(backplate))? detElem.child(_Unicode(backplate)).attr<double>(_Unicode(thickness)) : 0.;
```
Comment (wdconinc) [inline src/InsertCalorimeter_geo.cpp:52]: This looks a bit janky (ternary operator is not very readable). I assume there's somewhere else another `if` to then place the backplate or not.

Comment (rymilton) [reply src/InsertCalorimeter_geo.cpp:52]: Ah, no there's not actually. The backplate is included in the layer section that's placed via the existing for loop. But I need the thickness of the final layer (which is the backplate) before the for loop to make a function that calculates the radius and position of the beampipe hole.

Comment (rymilton) [reply src/InsertCalorimeter_geo.cpp:52]: Err if that wasn't clear, the hole shape is determined by a linear interpolation from the front of the insert to the front of the final layer. So I need the (total length - final layer length) to calculate the hole info.

Comment (wdconinc) [reply src/InsertCalorimeter_geo.cpp:52]: Nah, this is fine. Maybe just add a comment here to clarify.

**Review by wdconinc — APPROVED** [review]

**Comment by wdconinc** [comment]
Restarted failing benchmarks after benchmarks bug fix.

**Pull request merged**